jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/transport/HttpClientTransportOverHTTP.java

```diff
@@ -26,11 +26,12 @@
 import org.eclipse.jetty.io.EndPoint;
 import org.eclipse.jetty.util.annotation.ManagedAttribute;
 import org.eclipse.jetty.util.annotation.ManagedObject;
+import org.eclipse.jetty.util.thread.Invocable;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 @ManagedObject("The HTTP/1.1 client transport")
-public class HttpClientTransportOverHTTP extends AbstractConnectorHttpClientTransport
+public class HttpClientTransportOverHTTP extends AbstractConnectorHttpClientTransport implements Invocable
```

Comment:
The `HttpClientTransport` super-interface already extends from `Invocable`.

Reply:
Done.